src/components/utils/test/message_queue_test.cc

```diff
@@ -209,7 +209,7 @@ TEST_F(MessageQueueTest,
   threads::Thread* thread1 =
       threads::CreateThread("test thread", new ShutDownQueueDelegate(job_sync));
   thread1->start();
-  job_sync.WaitJobDone(100);
+  job_sync.WaitJobDone(2000);
```

Comment:
@LuxoftAKutsan, more simplified version in [1fb7d45](https://github.com/smartdevicelink/sdl_core/pull/707/commits/1fb7d45870bfeab6926d2d7da6857a36028bdb0b).